libs/nss/Makefile

```diff
@@ -0,0 +1,108 @@
+#
+# Copyright (C) 2017 Lucian Cristian
+#
+# This is free software, licensed under the GNU General Public License v2.
+# See /LICENSE for more information.
+#
+include $(TOPDIR)/rules.mk
+
+PKG_NAME:=nss
+PKG_VERSION:=3.42.1
+PKG_RELEASE:=1
+PKG_MAINTAINER:=Lucian Cristian <lucian.cristian@gmail.com>
+PKG_LICENCE:=GPL-2.0+
+
+PKG_SOURCE:=$(PKG_NAME)-$(PKG_VERSION)-with-nspr-4.20.tar.gz
+PKG_SOURCE_URL:=https://ftp.mozilla.org/pub/security/nss/releases/NSS_3_42_1_RTM/src/
```

Comment:
Please use 
https://download.cdn.mozilla.net/pub/security/nss/releases/NSS_3_42_1_RTM/src/ and 
https://archive.mozilla.org/pub/security/nss/releases/NSS_3_42_1_RTM/src/ instead